open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -13,10 +13,13 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def get_diff(df, column_name):
</pre><p>This is too simple to be a function. Let's just not have it.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>DONE</p>